src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -471,9 +472,7 @@ public void cannotCreateARequestWithInvalidStatus(String status)
       response, hasStatus(HTTP_BAD_REQUEST));
 
     assertThat(response.getBody(),
-      is("Request status must be \"Open - Not yet filled\", " +
-        "\"Open - Awaiting pickup\", \"Open - In transit\", " +
-        "\"Closed - Filled\", \"Closed - Unfilled\" or \"Closed - Pickup expired\""));
+      containsString("Request status must be one of the following:"));
```

Comment:
Why have these assertions been changed to be something less specific?

Reply:
Testing for exact error message makes tests more brittle, if a status is added in the future then this test will fail, the test is named cannotCreateARequestWithInvalidStatus and new assertion still checks that invalid status produces an error with message that tells user that status is invalid. I don't have strong opinions about this, I can change it to check for exact error message

Reply:
Without testing the exact message somewhere, how do we know that it includes the correct statuses?

I'm ok with the assertion being changed in the API tests if we have another way to demonstrate this message and the set of value request statuses. These could be unit tests for the the message and for whether a status is valid or not.

Reply:
An alternative is to extract a matcher out of the current tests, so that the message is only defined in a single place in the tests

Reply:
I agree that it needs to be tested somewhere, added check for full message in one test